RadixWallet/Prelude/Extensions/Collection+Extra.swift

```diff
@@ -95,3 +95,9 @@ extension Sequence {
 		try Dictionary(grouping: self, by: value)
 	}
 }
+
+extension Set {
```

Comment:
When/where are we using `Set` - I prefer if we used `OrderedSet` to make debugging and testing easier (I hate unordered-ness)

Reply:
Ah it is when you want to just check members, and ignore order, I see.